management/src/main/java/org/kaazing/gateway/management/config/ServiceConfigurationBeanImpl.java

```diff
@@ -216,14 +222,16 @@ public String getConnects() {
     public String getConnectOptions() {
         if (serviceContext.supportsConnects()) {
             ConnectOptionsContext context = serviceContext.getConnectOptionsContext();
+            Map<String, Object> connectOptions = context.asOptionsMap();
 
             JSONObject jsonOptions = new JSONObject();
 
             try {
                 if (context != null) {
```

Comment:
context will not be null at this line (as it already dereferenced at line 225), so you could eliminate null check